target/linux/ath79/image/generic.mk

```diff
@@ -752,6 +752,18 @@ define Device/dlink_dir-859-a1
 endef
 TARGET_DEVICES += dlink_dir-859-a1
 
+define Device/dlink_dir-869-a1
+  $(Device/seama)
+  SOC := qca9563
+  DEVICE_VENDOR := D-Link
+  DEVICE_MODEL := DIR-869
+  DEVICE_VARIANT := A1
+  IMAGE_SIZE := 15872k
+  DEVICE_PACKAGES :=  kmod-ath10k-ct-smallbuffers ath10k-firmware-qca988x-ct
+  SEAMA_SIGNATURE := wrgac54_dlink.2015_dir869
```

Comment:
Please add the line
`  SUPPORTED_DEVICES += dir-869-a1`
here, since this device has been supported in ar71xx, but the format of device names was changed for ath79.

This will add the old model string from ar71xx to the metadata appended to `.sysupgrade` images, allowing routers currently running ar71xx to update smoothly to ath79.

Just updated mine from LEDE, great to see this device ported to ath79! :slightly_smiling_face: